src/main.rs

```diff
@@ -60,10 +61,20 @@ fn main() -> Result<(), String> {
         Some((_, _)) => unreachable!(),
         None => {
             // Create the application.
-            let mut application = Application::new(matches.get_one::<String>("config").cloned())?;
+            let mut application =
+                match Application::new(matches.get_one::<String>("config").cloned()) {
+                    Ok(application) => application,
+                    Err(error) => {
+                        eprintln!("{error}");
+                        error!("{error}");
```

Comment:
I thought it could be nice to have the complete picture of what happened in the log, as well as a message to stderr to show the user that this was expected (not a crash) and why it quit.

Reply:
Hmm, does `log::error` log to `stdout` and `stderr`?

Reply:
I don't think it's set up to log to either. It only logs to the file if I remember correctly, because stdout/stderr wouldn't work during most of the program.

Reply:
Oh wow, right, this is a curses application 🤦